docs/concepts/sensor-models/sensor-models.md

```diff
@@ -0,0 +1,88 @@
+
+
+## The Community Sensor Model
+
+Because the only requirement of a sensor model is that it models a sensor's geometric information, the design and implementation of a sensor model largely remains open-ended.  Of particular importance is the sensor's interface, i.e. what information is required to create the model and what information is produced by the model.  Without a concrete interface specification, each sensor model may require and provide its own set of information, which makes it incredibly difficult to facilitate interoperability between sensor models. In order to address these concerns about interoperability, the US defense and intelligence community established the [Community Sensor Model (CSM) working group](https://gwg.nga.mil/gwg/focus-groups/Community_Sensor_Model_Working_Group_(CSMWG).html) , which is responsible for developing and maintaining a set of standards upon which sensor models can be implemented.  The resulting CSM standard describes an application programming interface (API) for multiple sensor types.
+
+Alone, a CSM is of little practical use. The standard defines an interface but does not define where necessary sensor metadata should be sourced, the format of said metadata, or the implementation of the photogrammetric algorithms used by the sensor. Therefore, a CSM exists within a broader ecosystem. First, some external capability or method must be used to generate image support data (ISD) that can be used to parameterize a CSM. Then, the CSM can parse and use the ISD to extract the parameters (interior and exterior orientation) necessary to perform photogrammetric operations (as defined by the CSM API). Finally, a SET is likely used to orchestrate the pieces of the CSM ecosystem and exploits said pieces in order to use the sensor for some desired operation [(Laura et al., 2020)](https://doi.org/10.1029/2019EA000713)
+
+!!! INFO "CSM -- Quick Definition"
+    The CSM is a standard that defines an API for sensor models, thereby providing a common structure that guarantees interoperability of any sensor models that adhere to the standard.
+
+## The ASC Sensor Model Ecosystem
+The Astrogeology Science Center has created and maintains a suite of software packages that provide end-to-end processing for sensor models, including a metadata specification, CSM-compliant sensor model implementation, and tools to leverage those sensor models for scientific processing.
```

Comment:
Here, sensor model is being used to describe the entirety of the system, but above it was just the mathematical model (I think?). Does ASC really maintain software for end-to-end process for sensor models or are the sensor models one piece in the larger ecosystem?